.github/workflows/rds-plan.yml

```diff
@@ -0,0 +1,35 @@
+name: rds plan terraform
+
+on:
+  pull_request:
+    paths:
+      - terraform/services/api-rds/**
```

Comment:
This is correct but the directory files below are in has this reversed with "rds-api". Move the service to terraform/services/api-rds.